packages/jest/src/plugins/plugin.spec.ts

```diff
@@ -352,354 +354,14 @@ describe('@nx/jest/plugin', () => {
 
       const results = await createNodesFunction(
         ['proj/jest.config.js'],
-        { targetName: 'test' },
+        { targetName: 'test', disableJestRuntime },
         context
       );
 
       expect(results).toMatchSnapshot();
```

Comment:
Can you change this to `matchInlineSnapshot`? I want them both to actually assert against the same snapshot... not their own respective snapshots which happen to be the same at the moment.